.github/workflows/test.yaml

```diff
@@ -38,5 +38,6 @@ jobs:
         env:
           CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
         with:
+          coverageCommand: npm test
```

Comment:
Not sure if this is really needed as the `npm test` command has been already executed in the previous step and it may be redundant.